doc/modules/lda_qda.rst

```diff
@@ -37,68 +37,57 @@ flexible.
 only makes sense in a multiclass setting.
 
-This is implemented in
-:func:`discriminant_analysis.LinearDiscriminantAnalysis.transform`. The desired
-dimensionality can be set using the ``n_components`` constructor parameter.
-This parameter has no influence on
-:func:`discriminant_analysis.LinearDiscriminantAnalysis.fit` or
-:func:`discriminant_analysis.LinearDiscriminantAnalysis.predict`.
+This is implemented in the `transform` method. The desired dimensionality can
+be set using the ``n_components`` parameter. This parameter has no influence
+on the `fit` and `predict` methods.
 
 .. topic:: Examples:
 
     :ref:`sphx_glr_auto_examples_decomposition_plot_pca_vs_lda.py`: Comparison of LDA and PCA
     for dimensionality reduction of the Iris dataset
 
+.. _lda_qda_math:
+
 Mathematical formulation of the LDA and QDA classifiers
 =======================================================
 
 Both LDA and QDA can be derived from simple probabilistic models which model
 the class conditional distribution of the data :math:`P(X|y=k)` for each class
-:math:`k`. Predictions can then be obtained by using Bayes' rule:
+:math:`k`. Predictions can then be obtained by using Bayes' rule, for each
+training sample :math:`x \in \mathcal{R}^d`:
 
 .. math::
-    P(y=k | X) = \frac{P(X | y=k) P(y=k)}{P(X)} = \frac{P(X | y=k) P(y = k)}{ \sum_{l} P(X | y=l) \cdot P(y=l)}
+    P(y=k | x) = \frac{P(x | y=k) P(y=k)}{P(x)} = \frac{P(x | y=k) P(y = k)}{ \sum_{l} P(x | y=l) \cdot P(y=l)}
 
-and we select the class :math:`k` which maximizes this conditional probability.
+and we select the class :math:`k` which maximizes this posterior probability.
 
 More specifically, for linear and quadratic discriminant analysis,
-:math:`P(X|y)` is modeled as a multivariate Gaussian distribution with
+:math:`P(x|y)` is modeled as a multivariate Gaussian distribution with
 density:
 
-.. math:: P(X | y=k) = \frac{1}{(2\pi)^{d/2} |\Sigma_k|^{1/2}}\exp\left(-\frac{1}{2} (X-\mu_k)^t \Sigma_k^{-1} (X-\mu_k)\right)
+.. math:: P(x | y=k) = \frac{1}{(2\pi)^{d/2} |\Sigma_k|^{1/2}}\exp\left(-\frac{1}{2} (x-\mu_k)^t \Sigma_k^{-1} (x-\mu_k)\right)
 
 where :math:`d` is the number of features.
 
-To use this model as a classifier, we just need to estimate from the training
-data the class priors :math:`P(y=k)` (by the proportion of instances of class
-:math:`k`), the class means :math:`\mu_k` (by the empirical sample class means)
-and the covariance matrices (either by the empirical sample class covariance
-matrices, or by a regularized estimator: see the section on shrinkage below).
+QDA
+---
 
-In the case of LDA, the Gaussians for each class are assumed to share the same
-covariance matrix: :math:`\Sigma_k = \Sigma` for all :math:`k`. This leads to
-linear decision surfaces, which can be seen by comparing the
-log-probability ratios :math:`\log[P(y=k | X) / P(y=l | X)]`:
+According to the model above, the log of the posterior is:
```

Comment:
This could be extended a bit to explain where `p(x)` goes, and what is the notation `Cst`